src/test/scala/ch/epfl/bluebrain/nexus/tests/Keycloak.scala

```diff
@@ -0,0 +1,151 @@
+package ch.epfl.bluebrain.nexus.tests
+
+import java.util.regex.Pattern.quote
+
+import akka.http.javadsl.model.headers.HttpCredentials
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.Authorization
+import akka.http.scaladsl.unmarshalling.FromEntityUnmarshaller
+import akka.stream.Materializer
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient.UntypedHttpClient
+import ch.epfl.bluebrain.nexus.commons.test.Resources
+import ch.epfl.bluebrain.nexus.tests.Identity.{ClientCredentials, UserCredentials}
+import ch.epfl.bluebrain.nexus.tests.Optics._
+import com.typesafe.scalalogging.Logger
+import io.circe.Json
+import monix.bio.Task
+
+import scala.concurrent.duration._
+
+trait Keycloak
+
+object Keycloak extends Keycloak with Resources {
```

Comment:
functions in the object are blocking; is there a reason for that, or could we change them to return `IO[*, A]`?

I think the test setup should control the execution (call the runSyncUnsafe if necessary)